.golangci-kal.yml

```diff
@@ -39,13 +39,11 @@ linters-settings:
             isFirstField: Warn # Require conditions to be the first field in the status struct.
             usePatchStrategy: Forbid # Conditions should not use the patch strategy on CRDs.
             useProtobuf: Forbid # We don't use protobuf, so protobuf tags are not required.
-        # jsonTags:
```

Comment:
I think we should leave this section commented out so that we can easily see what configuration is available to us, rather than removing the comments. It may help us in the future when trying to work out why something isn't behaving in the way we might expect

Reply:
Fixed!